# DP: Don't bump the version when doing distro builds

git diff acda0027559c6112ea91af13fb32dd4bef8f0b8b 9288e0fae61b716ce75334be906283de8b18b787

--- a/bfd/ChangeLog
+++ b/bfd/ChangeLog
@@ -4,11 +4,6 @@
 
 2022-02-09  Nick Clifton  <nickc@redhat.com>
 
-	* development.sh (development): Set to true.
-	* version.m4 (BFD_VERSION): Set to 2.38.0.
-
-2022-02-09  Nick Clifton  <nickc@redhat.com>
-
 	2.38 release.
 	* development.sh (development): Set to false.
 	(experimental): Set to false.
--- a/bfd/development.sh
+++ b/bfd/development.sh
@@ -16,7 +16,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # Controls whether to enable development-mode features by default.
-development=true
+development=false
 
 # Indicate whether this is a release branch.
 experimental=false
--- a/bfd/version.m4
+++ b/bfd/version.m4
@@ -1 +1 @@
-m4_define([BFD_VERSION], [2.38.0])
+m4_define([BFD_VERSION], [2.38])
--- a/gas/ChangeLog
+++ b/gas/ChangeLog
@@ -199,11 +199,6 @@
 
 2021-07-03  Nick Clifton  <nickc@redhat.com>
 
-	* configure: Regenerate.
-	* po/gas.pot: Regenerate.
-
-2021-07-03  Nick Clifton  <nickc@redhat.com>
-
 	* 2.37 release branch created.
 
 2021-07-02  Alan Modra  <amodra@gmail.com>
--- a/ld/ChangeLog
+++ b/ld/ChangeLog
@@ -182,11 +182,6 @@
 
 2021-07-03  Nick Clifton  <nickc@redhat.com>
 
-	* configure: Regenerate.
-	* po/ld.pot: Regenerate.
-
-2021-07-03  Nick Clifton  <nickc@redhat.com>
-
 	* 2.37 release branch created.
 
 2021-06-22  Nelson Chu  <nelson.chu@sifive.com>
--- a/opcodes/ChangeLog
+++ b/opcodes/ChangeLog
@@ -100,11 +100,6 @@
 
 2021-07-03  Nick Clifton  <nickc@redhat.com>
 
-	* configure: Regenerate.
-	* po/opcodes.pot: Regenerate.
-
-2021-07-03  Nick Clifton  <nickc@redhat.com>
-
 	* 2.37 release branch created.
 
 2021-07-02  Alan Modra  <amodra@gmail.com>
